test/functional/apps/dashboard_elements/controls/control_group_chaining.ts

```diff
@@ -108,12 +107,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('Selecting an option in the first Options List will filter the second and third controls', async () => {
       await dashboardControls.optionsListOpenPopover(controlIds[0]);
-      await dashboardControls.optionsListPopoverSelectOption('cat');
+      await dashboardControls.optionsListPopoverSelectOptions('cat');
       await dashboardControls.optionsListEnsurePopoverIsClosed(controlIds[0]);
 
       await dashboardControls.ensureAvailableOptionsEqual(controlIds[1], {
-        suggestions: { Tiger: 6, sylvester: 5 },
-        invalidSelections: [],
+        Tiger: 6,
+        sylvester: 5,
```

Comment:
I cleaned up the logic for `ensureAvailableOptionsEqual` so that it better aligns with its name - i.e. it previously would also check for invalid selections but, in most cases, this was just an empty array anyway. So, I made `ensureAvailableOptionsEqual` **only** check for the available options and  separated out the invalid selections check to be a separate function `ensureInvalidSelectionsEqual`, which I think is a lot cleaner 💃